packages/services/src/services/Profile/Profile.js

```diff
@@ -1,18 +1,17 @@
 /**
- * Copyright IBM Corp. 2020
+ * Copyright IBM Corp. 2020, 2021
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-import jsonp from 'jsonp';
+import axios from 'axios';
 
 /**
  * @constant {string | string} Host for the profile status API call
  * @private
  */
 const _host =
-  (process && process.env.PROFILE_HOST) || 'https://idaas.iam.ibm.com';
+  (process && process.env.PROFILE_HOST) || 'https://prepiam.ice.ibmcloud.com';
```

Comment:
we should actually set this as prod. We can test the preprod environment with PROFILE_HOST.

Reply:
`https://login.ibm.com` I believe